src/xenia/kernel/objects/xthread.h

```diff
@@ -156,6 +160,8 @@ class XThread : public XObject {
   uint32_t pcr_address_;
   uint32_t thread_state_address_;
   cpu::ThreadState* thread_state_;
+  bool guest_thread_;  // Launched into guest code?
```

Comment:
Didn't mean that to sound like it's fuzzy, it just literally means the thread was launched straight into guest code.